kore/src/Kore/Strategies/Goal.hs

```diff
@@ -171,6 +171,7 @@ allPathStrategy claims axioms =
             , RemoveDestination
             , TriviallyValid
             , DerivePar axioms
+            , RemoveDestination
```

Comment:
I think it's fine to remove (or to not add back) this one. I think you can also remove the similar `RemoveDestination` from `nextStep`.